controllers/chat/sendAnonymousMessage.js

```diff
@@ -67,6 +68,8 @@ const sendAnonymousMessage = async (req, res) => {
     if (createdChannel?.channel?.is_channel_blocked)
       return res.status(403).json(responseError('Channel is blocked'));
 
+    let channelMember = [];
+    let senderInfo = [];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variables `channelMember` and `senderInfo` are declared as arrays, but later in the code, `senderInfo` is assigned an object (line 90). This could lead to confusion and potential bugs. It would be better to initialize `senderInfo` as `null` or `{}`.
```diff
- let channelMember = [];
- let senderInfo = [];
+ let channelMember = [];
+ let senderInfo = null;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->